transmart-core-db-tests/src/groovy/org/transmartproject/db/dataquery/highdim/snp_lz/SnpLzTestData.groovy

```diff
@@ -208,6 +208,16 @@ class SnpLzTestData {
                         it.chromosome == annotation.chromosome }
             assert snpInfo != null
 
+            // Choose value of minorAllele based on the allele count, to vary its
+            // value: to generate test cases for both 'A1' and 'A2'.
+            long a1Count = 0
+            long a2Count = 0
+            gtss.each {
+                a1Count += it.findAll( a1 ).size()
+                a2Count += it.findAll( a2 ).size()
```

Comment:
Okay, I'll have another look.
But as far as I understand, minor alleles are not always alternate alleles.